app/src/main/java/edu/uc/forbesne/shoppingsidekick/ui/main/ProductListAdapter.kt

```diff
@@ -72,7 +72,8 @@ class ProductListAdapter( private val productList: ArrayList<Product>, mainViewM
         val btnIncrease: ImageButton = view_popup.findViewById(R.id.btnIncrease)
         val lblQuantity: TextView = view_popup.findViewById(R.id.etnQuantity)
 
-        Picasso.get().load(imgURL).into(prodImg);
+        val Picasso = null
```

Comment:
I updated the Picasso object and made it a val through the error that was shown on Android Studio and through suggestions on: https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements

Reply:
Ditto comments above.